### DISCUSSION TITLE
Combine PHP to Js

### DISCUSSION THREAD

**Discussion by WaWprjct** [community/community#134974] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How can Get PHP value to Js?!

**Comment by mwahyudihd** [comment, marked as answer]
It can Use DOM with function
``
<input type="text" id-data="<?= $data['id'] ?>" onclick="getId(this)">
``

```
const getId = function(element) {
 const dataValue = element.getAttribute("id-data");
#code
 return #code
}
```

**Comment by mdhifta** [comment]
Ideally you can use DOM. Create an input with a value and an id, then get the value with the DOM. For example:

-- PHP input value --
`<input type="text" value="<?= $value; ?>" id="get-php-value">`

-- JS get value -- 
```
<script>
 var data_value = document.getElementById("get-php-value").value;
</script>
```

but sometimes I often force myself and use bad methods such as:
```
<script>
 var data_value = <?= $value; ?>;
</script>
```
